src/platform/icelake/include/platform/platform.h

```diff
@@ -182,6 +182,8 @@ static inline void platform_panic(uint32_t p)
 #define platform_trace_point(__x) \
 	mailbox_sw_reg_write(SRAM_REG_FW_TRACEP, (__x))
 
+#define PLATFORM_XTAL_OSCILLATOR_CLK 38400000
+
```

Comment:
I thought ICL had support for cardinal frequencies? My understanding was that only SKL/KBL/CNL/CML did not have the required PLL